initialise_server.sql

```diff
@@ -29,6 +29,13 @@ CREATE TABLE IF NOT EXISTS playing_in (
     FOREIGN KEY (game_id) REFERENCES games(id)
 );
 
+-- Note that only 'tax' uses the value field in this table
+CREATE TABLE IF NOT EXISTS miscellaneous (
+    board_position tinyint UNSIGNED NOT NULL,
+    type ENUM('tax', 'chance', 'community_chest', 'jail', 'parking', 'to_jail') NOT NULL,
+    value smallint UNSIGNED,
+    PRIMARY KEY (board_position)
```

Comment:
We could consider having a cards table holding the chance and community chest cards having a **id** as a primary key for card uniqueness, **string** col for the text displayed on the card, **type** enum if it's a chance or community chest card and **action** could represent a case number for a particular action (increase player bal. by x amount, roll again... etc.)

Reply:
Yeah definitely, whoever is doing Task 8.1 should take this into account.